tests/maintel/test_warmup_hexapod.py

```diff
@@ -0,0 +1,125 @@
+):
+    def setUp(self):
+        pass
+
+    async def basic_make_script(self, index):
+        logger.debug("Starting basic_make_script")
+        self.script = WarmUpHexapod(index=index)
+
+        # self.script.checkpoints_activities = [
+        #     (checkpoint, unittest.mock.AsyncMock())
+        #     for checkpoint, _ in self.script.checkpoints_activities
+        # ]
+
+        logger.debug("Finished initializing from basic_make_script")
+        # Return a single element tuple
+        return (self.script,)
+
+    async def test_configure(self):
+        async with self.make_script():
+
+            # Try configure with minimum set of parameters declared
+            # Note that all are scalars and should be converted to arrays
+            hexapod = "camera_hexapod"
+            axis = "z"
+            step_size = 250
+            sleep_time = 2.0
+            max_position = 13000
+
+            await self.configure_script(
+                hexapod=hexapod,
+                axis=axis,
+                step_size=step_size,
+                sleep_time=sleep_time,
+                max_position=max_position,
+            )
+
+            assert self.script.config.hexapod == hexapod
+            assert self.script.config.axis == axis
+            assert self.script.config.step_size == [
+                step_size
+            ]  # int/float are converted to lists
+            assert self.script.config.sleep_time == [
+                sleep_time
+            ]  # int/float are converted to lists
+            assert self.script.config.max_position == max_position
+
+            assert self.script.hexapod_name == hexapod
+            assert self.script.sal_index == 1  # camera_hexapod
+
+    async def test_run(self):
+
+        # Start the test itself
+        async with self.make_script():
+
+            # Try configure with minimum set of parameters declared
+            # Note that all are scalars and should be converted to arrays
+            hexapod = "camera_hexapod"
+            axis = "z"
+            step_size = 2000
+            sleep_time = 0.1
+            max_position = 13000
+
+            # Configure the script
+            await self.configure_script(
+                hexapod=hexapod,
+                axis=axis,
+                step_size=step_size,
+                sleep_time=sleep_time,
+                max_position=max_position,
+            )
+            assert self.script.state.state == Script.ScriptState.CONFIGURED
+
+            # Add some mocks
+            class MockPosition:
+                position = [0.0, 0.0, 0.0, 0.0, 0.0, 0.0]
+
+            self.script.move_hexapod = unittest.mock.AsyncMock()
+            self.script.hexapod.tel_application.aget = unittest.mock.AsyncMock(
+                return_value=(MockPosition)
+            )
+
+            # Run the script
+            await self.run_script()
+            assert self.script.state.state == Script.ScriptState.DONE
```

Comment:
Can you add a `test_executable` test? Check this one https://github.com/lsst-ts/ts_standardscripts/blob/e9b64618b318c9c0dfdba950614c469949af630e/tests/test_run_command.py#L131-L134

Reply:
Done.